protocol/linkpreview/linkpreview_test.go

```diff
@@ -88,106 +155,130 @@ func TestGetLinks(t *testing.T) {
-				Hostname:    "github.com",
-				Title:       "Allow users to customize links · Issue #15469 · status-im/status-mobile",
-				URL:         "https://github.com/status-im/status-mobile/issues/15469",
-				Thumbnail: common.LinkPreviewThumbnail{
-					Width:   1200,
-					Height:  600,
-					URL:     "",
-					DataURI: "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAABLAAAAJYCAYAAABy",
-				},
-			},
-		},
-		{
-			url: "https://www.imdb.com/title/tt0117500/",
-			expected: common.LinkPreview{
-				Description: "The Rock: Directed by Michael Bay. With Sean Connery, Nicolas Cage, Ed Harris, John Spencer. A mild-mannered chemist and an ex-con must lead the counterstrike when a rogue group of military men, led by a renegade general, threaten a nerve gas attack from Alcatraz against San Francisco.",
-				Hostname:    "www.imdb.com",
-				Title:       "The Rock (1996) - IMDb",
-				URL:         "https://www.imdb.com/title/tt0117500/",
-				Thumbnail: common.LinkPreviewThumbnail{
-					Width:   1000,
-					Height:  1481,
-					URL:     "",
-					DataURI: "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkJCgg",
-				},
-			},
-		},
-		{
-			url: "https://www.youtube.com/watch?v=lE4UXdJSJM4",
-			expected: common.LinkPreview{
-				URL:         "https://www.youtube.com/watch?v=lE4UXdJSJM4",
-				Hostname:    "www.youtube.com",
-				Title:       "Interview with a GNU/Linux user - Partition 1",
-				Description: "GNU/Linux Operating SystemInterview with a GNU/Linux user with Richie Guix - aired on © The GNU Linux.Programmer humorLinux humorProgramming jokesProgramming...",
-				Thumbnail: common.LinkPreviewThumbnail{
-					Width:   1280,
-					Height:  720,
-					DataURI: "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAUDBA8",
-				},
-			},
+func readAsset(t *testing.T, filename string) []byte {
+	b, err := ioutil.ReadFile("../../_assets/tests/" + filename)
+	require.NoError(t, err)
+	return b
+}
+
+func Test_UnfurlURLs_YouTube(t *testing.T) {
+	url := "https://www.youtube.com/watch?v=lE4UXdJSJM4"
+	thumbnailURL := "https://i.ytimg.com/vi/lE4UXdJSJM4/maxresdefault.jpg"
+	expected := common.LinkPreview{
+		URL:         url,
+		Hostname:    "www.youtube.com",
+		Title:       "Interview with a GNU/Linux user - Partition 1",
+		Description: "GNU/Linux Operating SystemInterview with a GNU/Linux user with Richie Guix - aired on © The GNU Linux.Programmer humorLinux humorProgramming jokesProgramming...",
+		Thumbnail: common.LinkPreviewThumbnail{
+			Width:   1,
+			Height:  1,
+			DataURI: "data:image/webp;base64,UklGRiQAAABXRUJQVlA4IBgAAAAwAQCdASoBAAEAAQAaJaQAA3AA/vpMgAA",
 		},
 	}
 
-	var urls []string
-	for _, e := range examples {
-		urls = append(urls, e.url)
+	disabledStubs := false
+	if disabledStubs {
+		expected.Thumbnail.Width = 1280
+		expected.Thumbnail.Height = 720
+		expected.Thumbnail.DataURI = "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAUDBA8"
 	}
```

Comment:
`disabledStubs` is always false, the if statement can never be true.

Reply:
Indeed @Samyoul, I used this flag during development, as a way of quickly switching on/off between stubbed and real HTTP calls. Since it's causing some confusion, I'll remove this.